renovate.json

```diff
@@ -80,25 +81,15 @@
       "groupName": "typescript-eslint",
       "matchPackagePatterns": "@typescript-eslint/*"
     },
-    {
-      "groupName": "@types/node",
-      "matchPackageNames": ["@types/node"],
-      "allowedVersions": "< 21.0.0"
-    },
     {
       "groupName": "bundling fixtures",
       "matchPaths": ["test/bundling/fixtures/**/package.json"],
       "schedule": "every 6 months on the first day of the month"
     },
     {
       "groupName": "node",
-      "matchPackageNames": ["node", "cimg/node", "actions/setup-node"],
-      "enabled": false
-    },
-    {
-      "groupName": "examples",
-      "matchPaths": ["examples/**/package.json"],
-      "enabled": false
+      "matchPackageNames": ["@types/node", "node", "cimg/node", "actions/setup-node"],
+      "allowedVersions": "20"
```

Comment:
The current LTS is v22.
Have you considered bumping?
`mui-x` uses v22 on CI. 🤔